ui: tooltip component storybook

vladlos opened the pull request:
added storybook for testing tooltip html content layouts.

Resolves: #50094

Release note (ui): none

![Screenshot 2020-07-07 at 17 19 33](https://user-images.githubusercontent.com/12850886/86796965-ccedd000-c077-11ea-8424-03b0357ce07f.png)

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/51063)
<!-- Reviewable:end -->

elkmaster commented:
yarn vendor pr: https://github.com/cockroachdb/yarn-vendored/pull/23

vladlos committed:
ui: tooltip component storybook

added storybook for testing tooltip html content layouts.

Resolves: #50094

Release note (ui): none

koorosh commented on pkg/ui/src/components/tooltip/tooltip.tsx:
```diff
@@ -18,16 +18,18 @@ import "./tooltip.styl";
 export interface TooltipProps {
   children: React.ReactNode;
   theme?: "default" | "blue";
+  visible?: boolean;
```
Our <Tooltip> component inherits props from AntTooltip (`TooltipProps & AntTooltipProps`) so it is already has `visible` and `defaultVisible` props. So it doesn't require making any changes here at all.

koorosh commented on pkg/ui/src/components/tooltip/tooltip.stories.tsx:
```diff
@@ -0,0 +1,50 @@
+    "withKnobs",
+    () => (
+      <Tooltip
+        visible={boolean("Visible", true)}
```
`<Tooltip>` component inherits and propagates `defaultVisible` prop to AntTooltip. (see comment below)
I suggest using `defaultVisible` to avoid confusion in names because it also has another `visible` prop.

koorosh reviewed (changes requested):
The build is failing with error:
```
error Can't make a request in offline mode ("https://registry.yarnpkg.com/@storybook/addon-knobs/-/addon-knobs-5.3.19.tgz")
```
Probably not all packages were added to `yarn-vendor`?
